game/src/debug/routes.rs

```diff
@@ -208,7 +208,7 @@ fn params_to_controls(ctx: &mut EventCtx, mode: TripMode, params: &RoutingParams
             Spinner::widget(
                 ctx,
                 (1, 100),
-                (params.unprotected_turn_penalty * 10.0) as isize,
+                (params.unprotected_turn_penalty.inner_seconds() * 10.0) as isize,
```

Comment:
I'm not familiar with this code - but it looks like here, though we're storing seconds, we're not actually expressing the penalty as seconds. We transform some number to seconds, then have to remember to untransform it elsewhere.

Rather than transforming, and remembering to untransform, would it be reasonable to directly express the penalty as seconds in the UI?

(feel free to ignore this suggstion - i'm mostly just trying to test my understanding of your overall goal here)

Reply:
Your understanding is correct. The problem is that `Spinner` only understands `isize`s. The multiplication by 10 was to allow setting centi-second granularity. But actually, that level of precision is no longer useful, now that `unprotected_turn_penalty` is added and not multiplied. I'm removing the *10.

Later on, the spinner widget ought to be generic and understand `Duration`, `Distance`, and other types natively.